client/src/components/Module.jsx

```diff
@@ -30,6 +32,52 @@ function Module(props) {
 
   useEffect(() => { setImageURL(link); }, [link]);
 
+  const deleteModule = async (moduleId) => { // calls api to delete modules, then removes that module from state children array in ExpandedModule
+    api.deleteModule(moduleId).then(() => {
+      // reloads the page
+      deleteChild(moduleId);
+    });
+  };
+  if (role === 'admin') {
```

Comment:
Do you think you can put this into the return statement so we just have one return?